### PR TITLE
drag drop table hover

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- fixed hover flickering of borders
- fixes hover colors of dropped element (body, table or row)

### CODE REVIEW

File: filer/static/filer/js/addons/table-dropzone.js [hunk 1]
```diff
@@ -119,10 +119,20 @@
...
+                            bottom: dropzoneBoundingRect.bottom,
+                            width: dropzoneBoundingRect.width,
+                            height: dropzoneBoundingRect.height
+                        };
+
                         $(dropzones).addClass('reset-hover');
                         uploadSuccess.addClass(hiddenClass);
                         infoMessage.removeClass(hiddenClass);
                         dropzone.addClass(dragHoverClass).removeClass('reset-hover');
+                        $('.drag-hover-border').show().css(dropzonePosition);
```
Comment (vxsx) [inline filer/static/filer/js/addons/table-dropzone.js]: if you swap call to `css` with a call to `show` you will get one less repaint

**Comment by Brankub** [comment]
Looks good, but let's wait for @vxsx or @FinalAngel review too.

File: filer/static/filer/js/addons/table-dropzone.js (hunk 1 above)
Comment (lory87) [inline filer/static/filer/js/addons/table-dropzone.js]: @vxsx yes its not needed removed display none and show

Comment (vxsx) [inline filer/static/filer/js/addons/table-dropzone.js]: didn't push?

**Comment by vxsx** [comment]
👍

**Comment by FinalAngel** [comment]
LGTM

**Pull request merged**